hack/generator/cmd/gen/gen.go

```diff
@@ -33,7 +33,10 @@ func NewGenCommand() (*cobra.Command, error) {
 			configuration, err := loadConfiguration("azure-cloud.yaml")
 			if err != nil {
 				log.Printf("Error loading configuration: %v\n", err)
-				writeSampleConfig("public-cloud-sample.yaml")
+				err2 := writeSampleConfig("public-cloud-sample.yaml")
+				if err2 != nil {
+					log.Printf("Failed to write sample configuration file, err: %v\n", err2)
+				}
```

Comment:
I removed the sample file writing in my logging PR

Reply:
Cool - for now just to have it pass lint will leave this - if you merge first then I'll fix it in conflict resolution otherwise you'll have to.